### ISSUE TITLE
Ensure game is keyboard accessible

### ISSUE THREAD

**Issue by svillegascreative** [svillegascreative/vue-bingo#6]
## Problem
Boxes cannot currently be navigated or selected with the keyboard.

**Comment by svillegascreative** [comment]
I think `display: none;` on checkboxes is at the root. Checkboxes are removed from DOM, thus cannot be accessed. Need to read on a better hack to hide them (how to make accessible custom checkboxes). Also ensure label has a visible `:focus` state.

**Comment by svillegascreative** [comment]
Keyboard navigation has been fixed and focus style added.
Focus management problem remains: when the game is won, all checkboxes become disabled and focus is lost. It should be restored to the controls (clear board button, maybe?)
